src/components/Header/Header.js

```diff
@@ -1,26 +1,95 @@
 import React, { useState } from 'react'
 import { FaTimes, FaBars } from 'react-icons/fa';
-import { HeaderContainer, Logo, MobileIcon, MobileMenu } from './Header.elements'
+import { IconContext } from 'react-icons/lib';
+import { 
+  HeaderContainer, 
+  HeaderWrapper, 
+  Logo, 
+  LogoIcon, 
+  MobileIcon, 
+  NavMenu,
+  NavItem, 
+  NavLinks,
+  UserOptions,
+  UserIcon
+} from './Header.elements'
 
 
 const Header = () => {
-  const [click, setClick] = useState(false)
+  const [mobileNav, setMobileNav] = useState(false)
+  const handleMobileNav = () => {
+    setUserDropdown(false) 
+    return setMobileNav(!mobileNav)
+  }
+   
+  const [userDropdown, setUserDropdown] = useState(false)
```

Comment:
I like the way this is used to control the dropdown for the user profile button, it might be helpful to rename it "userDropdownOpen" for better readability/clarity on what this is doing